ishell/console.py

```diff
@@ -8,11 +8,11 @@
 
 
 class Console(object):
-    def __init__(self, prompt="Prompt", prompt_delim=">", welcome_message=None):
+    def __init__(self, prompt="Prompt", prompt_delim=">", exit_on_sigint=True):
         self.childs = {}
         self.prompt = prompt
         self.prompt_delim = prompt_delim
-        self.welcome_message = welcome_message
+        self.exit_on_sigint = exit_on_sigint
```

Comment:
I think welcome_message is a new attribute, coming from upstream and exit_on_sigint is an addition of ours. If this is correct, we should keep both.